src/Config/Dialog/ImportConfigDialog.ts

```diff
@@ -0,0 +1,50 @@
+/*
```

Comment:
Can you provide how `media/Config/importConfig.js` and this file are related?
`oneImportTools` nor `oneOtherTools` are not used here.

Reply:
ImportConfigDialog.ts let users to choose .cfg file which they want to import.
and then all values of .cfg file sended to receiveFromPanel.js which is not uploaded yet.
finally, importConfig.js is used in receiverForm.js. methods in importConfig.js show values of .cfg file in webview

Reply:
If adding each file in separate PR is possible can you split the PR ?

Reply:
you mean ImportConfigDialog.ts and importConfig.js shoud be in seperate PR?

Reply:
yup, if possible, if not can you share why?

Reply:
okay. I will make another PR for ImportConfigDialog.ts